cmd/libs/go2idl/go-to-protobuf/protobuf/cmd.go

```diff
@@ -137,6 +138,7 @@ func Run(g *Generator) {
 
 	protobufNames := NewProtobufNamer()
 	outputPackages := generator.Packages{}
+	nonOutputPackages := sets.NewString()
```

Comment:
If you'd prefer to avoid the apimachinery import, I can just make this a `map[string]struct{}`